### ISSUE TITLE
pilot description length error and stuck on waiting for iTunes 

### ISSUE THREAD

**Issue by yotavm** [fastlane/fastlane#11534]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
hi, 
when Fastlane start pilot i get description length error 
after that it looks like all good but its stuck on waiting for iTunes 
(using xcode 9.2 on ios 10.12)

##### Complete output when running Fastlane, including the stack trace and command, used
[[18:04:53]: Generated plist file with the following values:
[18:04:53]: ▸ -----------------------------------------
[18:04:53]: ▸ {
[18:04:53]: ▸   "provisioningProfiles": {
[18:04:53]: ▸     "org.personalnutritionproject": "Personalized Nutrition Project"
[18:04:53]: ▸   },
[18:04:53]: ▸   "method": "app-store",
[18:04:53]: ▸   "signingStyle": "manual"
[18:04:53]: ▸ }
[18:04:53]: ▸ -----------------------------------------
[18:04:53]: $ /usr/bin/xcrun /Users/yotav/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.75.1/gym/lib/assets/wrap_xcodebuild/xcbuild-safe.sh -exportArchive -exportOptionsPlist '/var/folders/0z/cpmk_pnx0xz_0c7h7hz0msm80000gn/T/gym_config20180111-14046-ay6ygf.plist' -archivePath /Users/yotav/Library/Developer/Xcode/Archives/2018-01-11/weizmann\ 2018-01-11\ 18.01.42.xcarchive -exportPath '/var/folders/0z/cpmk_pnx0xz_0c7h7hz0msm80000gn/T/gym_output20180111-14046-pswdv' -allowProvisioningUpdates 
[18:04:57]: Compressing 1 dSYM(s)
[18:04:57]: $ cd '/Users/yotav/Library/Developer/Xcode/Archives/2018-01-11/weizmann 2018-01-11 18.01.42.xcarchive/dSYMs' && zip -r '/Users/yotav/Desktop/500tech/weizmann-nutrition/weizmann.app.dSYM.zip' *.dSYM
[18:04:58]: ▸ updating: weizmann.app.dSYM/ (stored 0%)
[18:04:58]: ▸ updating: weizmann.app.dSYM/Contents/ (stored 0%)
[18:04:58]: ▸ updating: weizmann.app.dSYM/Contents/Info.plist (deflated 52%)
[18:04:58]: ▸ updating: weizmann.app.dSYM/Contents/Resources/ (stored 0%)
[18:04:58]: ▸ updating: weizmann.app.dSYM/Contents/Resources/DWARF/ (stored 0%)
[18:04:58]: ▸ updating: weizmann.app.dSYM/Contents/Resources/DWARF/weizmann (deflated 69%)

[18:04:58]: Successfully exported and compressed dSYM file
[18:04:58]: Successfully exported and signed the ipa file:
[18:04:58]: /Users/yotav/Desktop/500tech/weizmann-nutrition/weizmann.ipa
[18:04:58]: -------------------
[18:04:58]: --- Step: pilot ---
[18:04:58]: -------------------
[18:04:58]: Login to iTunes Connect (devapp@weizmann.ac.il)
[18:05:02]: Login successful
[18:05:03]: Ready to upload new build to TestFlight (App: 1290081421)...
[18:05:03]: Going to upload updated app to iTunes Connect
[18:05:03]: This might take a few minutes. Please don't interrupt the script.
[18:05:12]: [Transporter Error Output]: description length:722428
[18:06:17]: iTunes Transporter successfully finished its job
[18:06:17]: [iTMSTransporter] [2018-01-11 18:06:17 IST] <main> DBG-X: Apple's web service operation return value:

[18:06:17]: [iTMSTransporter] [2018-01-11 18:06:17 IST] <main> DBG-X:   parameter Success = true

[18:06:17]: [iTMSTransporter] [2018-01-11 18:06:17 IST] <main> DBG-X: Unlocking token at path /Users/yotav/.itmstransporter/UploadTokens/yotavs-MBP-2_WeizmannInstituteofScience_1290081421.itmsp.token

[18:06:17]: [iTMSTransporter] [2018-01-11 18:06:17 IST] <main> DBG-X: Releasing token file lock

[18:06:17]: [iTMSTransporter] [2018-01-11 18:06:17 IST] <main> DBG-X: Removing token at path /Users/yotav/.itmstransporter/UploadTokens/yotavs-MBP-2_WeizmannInstituteofScience_1290081421.itmsp.token

[18:06:17]: [iTMSTransporter] [2018-01-11 18:06:17 IST] <main>  INFO: Done performing uploadDone notification to Apple.

[18:06:17]: [iTMSTransporter] [2018-01-11 18:06:17 IST] <main>  INFO: The package: /var/folders/0z/cpmk_pnx0xz_0c7h7hz0msm80000gn/T/d20180111-14046-9nbymj/1290081421.itmsp has been successfully uploaded.

[18:06:17]: [iTMSTransporter] 

[18:06:17]: [iTMSTransporter] 

[18:06:17]: [iTMSTransporter] 

[18:06:17]: [iTMSTransporter] Package Summary:

[18:06:17]: [iTMSTransporter]  

[18:06:17]: [iTMSTransporter] 1 packages were uploaded successfully:

[18:06:17]: [iTMSTransporter]   /var/folders/0z/cpmk_pnx0xz_0c7h7hz0msm80000gn/T/d20180111-14046-9nbymj/1290081421.itmsp

[18:06:17]: [iTMSTransporter] [2018-01-11 18:06:17 IST] <main> DBG-X: Returning 0

[18:06:17]: iTunes Transporter output above ^
[18:06:17]: description length:722428
[18:06:17]: Although errors occurred during execution of iTMSTransporter, it returned success status.
[18:06:17]: --------------------------------------------------------------------
[18:06:17]: Successfully uploaded package to iTunes Connect. It might take a few minutes until it's visible online.
[18:06:17]: --------------------------------------------------------------------
[18:06:17]: Successfully uploaded the new binary to iTunes Connect
[18:06:17]: If you want to skip waiting for the processing to be finished, use the `skip_waiting_for_build_processing` option
[18:06:26]: Waiting for iTunes Connect to finish processing the new build (1.49 - 11)
[18:06:56]: Waiting for iTunes Connect to finish processing the new build (1.49 - 11)
[18:07:27]: Build doesn't show up in the build list anymore, waiting for it to appear again
[18:07:58]: Build doesn't show up in the build list anymore, waiting for it to appear again
[18:08:28]: Build doesn't show up in the build list anymore, waiting for it to appear again
[18:08:59]: Build doesn't show up in the build list anymore, waiting for it to appear again
[18:09:30]: Build doesn't show up in the build list anymore, waiting for it to appear again]

### Environment
[<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.12.6                                     |
| Ruby                        | 2.2.4                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.14.3 (Apple Git-98)           |
| Installation Source         | ~/.fastlane/bin/bundle/bin/fastlane         |
| Host                        | Mac OS X 10.12.6 (16G29)                    |
| Ruby Lib Dir                | ~/.fastlane/bin/bundle/lib                  |
| OpenSSL Version             | OpenSSL 1.0.2g  1 Mar 2016                  |
| Is contained                | false                                       |
| Is homebrew                 | true                                        |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 9.2                                         |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE | en_US.UTF-8 | ✅ |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# This file contains the fastlane.tools configuration
# You can find the documentation at https://docs.fastlane.tools
#
# For a list of all available actions, check out
#
#     https://docs.fastlane.tools/actions
#

# Uncomment the line if you want fastlane to automatically update itself
# update_fastlane

default_platform(:ios)

platform :ios do

  desc "Push a new beta build to TestFlight"
  
  lane :beta do
    
    increment_build_number(xcodeproj: "./ios/weizmann.xcodeproj")
    
    build_app(
      project: "./ios/weizmann.xcodeproj",
      scheme: "weizmann",
      export_xcargs: "-allowProvisioningUpdates"
    )

    # upload_to_testflight
    pilot

    clean_build_artifacts

    commit_version_bump(
      message: "👷 Pushed new build on TestFlight (build #{lane_context[SharedValues::BUILD_NUMBER]})",
      xcodeproj: './ios/weizmann.xcodeproj'
    )
  
  end

end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier "org.personalnutritionproject" # The bundle identifier of your app

# For more information about the Appfile, see:
#     https://docs.fastlane.tools/advanced/#appfile

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.75.1  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| slack-notifier            | 1.5.1        |
| CFPropertyList            | 2.3.5        |
| claide                    | 1.0.2        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.5.4        |
| rouge                     | 1.11.1       |
| xcpretty                  | 0.2.6        |
| terminal-notifier         | 1.7.1        |
| plist                     | 3.2.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.1        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.6.4        |
| tty-cursor                | 0.5.0        |
| tty-spinner               | 0.7.0        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.5        |
| excon                     | 0.55.0       |
| faraday                   | 0.12.1       |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.1.0        |
| gh_inspector              | 1.0.3        |
| json                      | 1.8.1        |
| mini_magick               | 4.5.1        |
| multi_json                | 1.12.1       |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 0.0.4        |
| dotenv                    | 2.2.0        |
| bundler                   | 1.14.6       |
| faraday_middleware        | 0.11.0.1     |
| uber                      | 0.0.15       |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 2.1.0        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| little-plugger            | 1.1.4        |
| logging                   | 2.2.2        |
| jwt                       | 1.5.6        |
| memoist                   | 0.15.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.1        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.13.6       |
| libxml-ruby               | 3.0.0        |
| unicode-display_width     | 1.1.3        |
| terminal-table            | 1.7.3        |
</details>

*generated on:* **2018-01-11**
</details>]

**Comment by mebowen** [comment]
I am also seeing this issue

**Comment by walsht** [comment]
So, I was getting `Build doesn't show up in the build list anymore, ...` today let it run for 4 hours.  I got the email that it finished processing after just an hour.  So, I opened another Terminal window and ran `fastlane pilot builds`  then magically the pilot process found the new build and finished sending to TestFlight successfully!

Maybe fastlane can run `fastlane pilot builds` when `Build doesn't show up in the build list anymore, ...` is returned?
